lib/parser/characters.h

```diff
@@ -26,7 +26,13 @@
 
 namespace Fortran::parser {
 
-enum class Encoding { UTF8, EUC_JP };
+// We can easily support Fortran program source in any character
+// set whose first 128 code points correspond to ASCII codes 0-127 (ISO/IEC646).
+// The specific encodings that we can handle include:
+//   LATIN_1: ISO 8859-1 Latin-1
+//   UTF_8: Multi-byte encoding of Unicode (ISO/IEC 10646)
+//   EUC_JP: 1-3 byte encoding of JIS X 0208 / 0212
+enum class Encoding { LATIN_1, UTF_8, EUC_JP };
```

Comment:
Should you use the ENUM_CLASS macro here?

Reply:
Nah.  I never need to map those enumeration values to strings.